db.json

```diff
@@ -80,6 +80,8 @@
     {
       "id": 1,
       "name": "Edeka",
+      "rating": "4",
```

Comment:
What's the advantage of saving the rating as string instead of number? Seems a bit counter-intuitive at first sight ;)

Reply:
Oh. Good point. I will have to change that :)